app/services/mongodb/MongoDBExtractionService.scala

```diff
@@ -118,7 +132,7 @@ class MongoDBExtractionService extends ExtractionService {
           }
           case None => {}
         }
-        groupings = groupings + (e.extractor_id -> ExtractionGroup(grp_start.toString, grp_end.toString, grp_endmsg, grp_list :+ e))
+        groupings = groupings + (e.extractor_id -> ExtractionGroup(grp_start.toString, grp_end.toString, grp_endmsg, grp_map))
```

Comment:
I think I've made a mistake in the logic for building up the ExtractionGroup Map here... the UI appears to only show one event from each job, and no longer lists all events related to the given jobId.

that last bit of syntax with the `:+` is probably some Scala shorthand to concatenate the new entry into the Map